### ISSUE TITLE
How to train the pre-trained model from scratch

### ISSUE THREAD

**Issue by xxLifeLover** [jiaowangswust/D2MVSC#1]
Thank you for your excellent work. 

I undoubtedly achieved the accuracy in the paper using the given pre-trained parameters. But how should I train the pre-trained model from scratch which will make it convenient to validate on other datasets.

Thanks again for sharing and looking forward to your reply!
